bedrock/mozorg/templates/mozorg/about/manifesto.html

```diff
@@ -52,9 +58,21 @@
 <main role="main">
   <article role="article" itemscope itemtype="http://schema.org/Article">
     <header id="main-feature">
-      <h1 itemprop="name">{{ self.page_title() }}</h1>
-      <p itemprop="description">{{ self.page_desc() }}</p>
+      <div class="content">
+        <h1 itemprop="name">{{ self.page_title() }}</h1>
+        <p itemprop="description">{{ self.page_desc() }}</p>
+      </div>
     </header>
+
+  {% if version == 'b' and switch('mozorg-search-activist') %}
+    <aside id="moz-search" class="moz-search moz-search-activist" data-terms="{{ _('Net Neutrality, Encryption, Surveillance, Copyright Law, Cybersecurity') }}">
+      <div class="content">
+        <label class="search-label" for="gsc-i-id1"><span class="search-label-prefix">{{ _('Search Mozilla:') }}</span> <span class="search-label-term">{{ _('Cybersecurity') }}</span></label>
+        <gcse:search gname="mozsearch"></gcse:search>
+      </div>
+    </aside>
```

Comment:
That's the plan, but I was waiting until I start building out the other two searches. I want to make it a macro that accepts a few arguments for a modifier class, suggested terms, etc. I suppose for now it could just be a static include though.